.eslintrc.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
         },
       },
     ],
+    'no-console': 'error',
```

Comment:
you can put it higher than other testing related rules (and even react specific one), because it concerns the main code.